contribs/gmf/src/sass/vars_only.scss

```diff
@@ -38,6 +38,7 @@ $hover-background-color: lighten($brand-primary, $standard-variation) !default;
 $brand-secondary-dark: darken($brand-secondary, $standard-variation) !default;
 
 $displayquerywindow-desktop-height: 15rem !default;
+$displayquerygrid-height: 15.62rem !default;
```

Comment:
I have not used the "desktop" keyword here because I don't know if it is really a desktop-only style. Should I?

Reply:
no :-)